src/components/PagesComponent/SoftwareDevelopers/style.js

```diff
@@ -88,7 +88,7 @@ const SoftwareDevelopersWrapper = styled.div`
           padding: 22px 48px;
         }
         p {
-          font-weight: 700;
+          font-weight: 700 !important;
```

Comment:
do not use important

Reply:
i am using 2 class at a same time and also both class nesting code are different thats why i have to add important if you have any suggestion then do let me know

Reply:
Give a className instead using direct html tag

Reply:
we are writing code in Sass not in CSS so are you this will not create any issue